.rubocop.yml

```diff
@@ -71,3 +68,40 @@ Style/StringLiterals:
 Style/ZeroLengthPredicate:
   Exclude:
     - 'lib/moab/bagger.rb' # is calling `x.size == 0` on Pathname objects, which don't have `.empty?` on old Rubies
+
+Layout/SpaceBeforeBrackets: # (new in 1.7)
+  Enabled: true
+Lint/AmbiguousAssignment: # (new in 1.7)
+  Enabled: true
+Lint/DuplicateBranch: # (new in 1.3)
+  Enabled: true
+Lint/DuplicateRegexpCharacterClassElement: # (new in 1.1)
+  Enabled: true
+Lint/EmptyBlock: # (new in 1.1)
+  Enabled: true
+Lint/EmptyClass: # (new in 1.3)
+  Enabled: true
+Lint/NoReturnInBeginEndBlocks: # (new in 1.2)
+  Enabled: true
+Lint/ToEnumArguments: # (new in 1.1)
+  Enabled: true
+Lint/UnexpectedBlockArity: # (new in 1.5)
+  Enabled: true
+Lint/UnmodifiedReduceAccumulator: # (new in 1.1)
+  Enabled: true
+Style/ArgumentsForwarding: # (new in 1.1)
+  Enabled: true
+Style/CollectionCompact: # (new in 1.2)
+  Enabled: true
+Style/DocumentDynamicEvalDefinition: # (new in 1.1)
+  Enabled: true
+Style/HashExcept: # (new in 1.7)
+  Enabled: true
+Style/NegatedIfElseCondition: # (new in 1.2)
+  Enabled: true
+Style/NilLambda: # (new in 1.3)
+  Enabled: true
+Style/RedundantArgument: # (new in 1.4)
+  Enabled: true
+Style/SwapValues: # (new in 1.1)
+  Enabled: true
```

Comment:
I find it easier to figure out what is going on when our rubocop.yml has split the rules into their uber-sections ...

Reply:
I'm just copying and pasting exactly what rubocop told me to paste in here.  If you like it formatted, perhaps consider a separate PR.

Reply:
I understand exactly how and why you did it;  I'm suggesting that for team maintenance purposes, it's more useful if these PRs include taking another 30 seconds to put the diff groups of new rules under their headings.  I don't want to chase all over a rubocop.yml file to find the rules and our settings -- I want some sort of organization to it, especially as it gets longer and longer.

Reply:
This seems a reasonable request to me with more upside than downside.